It seems we need to fix the dependence issues in `solidpod`, perhaps upgrading all the packages to their latest version if possible.

Currently, I got this when trying to run `keypod` using branch `zy/136_resolve_dependencies`
```
$ make linux
flutter run --device-id linux
Resolving dependencies... (1.5s)
Because every version of solidpod from path depends on package_info_plus ^5.0.1 and no versions of package_info_plus match >5.0.1 <6.0.0, every version of solidpod
  from path requires package_info_plus 5.0.1.
And because package_info_plus 5.0.1 depends on web >=0.3.0 <0.5.0, every version of solidpod from path requires web >=0.3.0 <0.5.0.
Because http >=1.2.2 depends on web >=0.5.0 <2.0.0 and http >=1.2.1 <1.2.2 depends on web ^0.5.0, http >=1.2.1 requires web >=0.5.0 <2.0.0.
Thus, solidpod from path is incompatible with http >=1.2.1.
And because rdflib >=0.2.10 depends on http ^1.2.1, solidpod from path is incompatible with rdflib >=0.2.10.
So, because keypod depends on both rdflib ^0.2.10 and solidpod from path, version solving failed.


You can try the following suggestion to make the pubspec resolve:
* Consider downgrading your constraint on rdflib: flutter pub add rdflib:^0.2.9
make: *** [support/flutter.mk:96: linux] Error 1
```

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -63,6 +63,8 @@ dependency_overrides:
   # override asap.
 
   intl: ^0.19.0
+  
+  package_info_plus: ^8.0.0
 
 # For information on the generic Dart part of this file, see the
 # following page: https://dart.dev/tools/pub/pubspec

